crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -178,7 +201,48 @@ pub(crate) async fn bench(
 
     client.stop().await;
 
+    let took = start.elapsed();
+
+    let space_allocated = get_size(base_directory)?;
+    let actual_space_pledged = multi_farming
+        .plots
+        .iter()
+        .map(Plot::piece_count)
+        .sum::<u64>()
+        * PIECE_SIZE as u64;
+    let overhead = space_allocated - actual_space_pledged;
```

Comment:
Well, we already know overhead because we only use 80% of space specified. Would be helpful to actually compute the size of each plot folder and show both actual overhead and space wasted on top of that (because we don't do actual estimates yet). Probably in a separate PR.

Reply:
I thought it would be helpful to print actual used space. Overall I also thought that we could do some benchmarking and remove this cap of 80% to some more precise formula for that. What do you think?

Reply:
Yes, we definitely want to use something more precise than 80%.

I actually missed `get_size` call above, so you are measuring correct space. I think the only parameter missing is wasted space. Namely you allocate 1G, 800M is used by plot, 100M by other databases, then 100M is wasted because it isn't used by anything.